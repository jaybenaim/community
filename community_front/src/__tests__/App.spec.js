import React from "react";
import { create } from "react-test-renderer";

// CHECKS IF COMPONENTS ALL GET RENDERED

describe("Nav Component", () => {
  test("App has a MyProfile component ", () => {
    const navbar = create(<nav class="nav navbaru"> </nav>);
    expect(navbar.toJSON()).toMatchSnapshot();
  });
});

describe("MyProfile Component", () => {
  test("App has a MyProfile component ", () => {
    const myProfile = create(<div class="cont container"></div>);
    expect(myProfile.toJSON()).toMatchSnapshot();
  });
});

describe("MyProfile Component", () => {
  test("App has a SearchPage component ", () => {
    const searchPage = create(<div class="container container-fluid"></div>);
    expect(searchPage.toJSON()).toMatchSnapshot();
  });
});

describe("MyProfile Component", () => {
  test("App has a Map component ", () => {
    const map = create(<section style="height: 100vh; width: 100%;"></section>);
    expect(map.toJSON()).toMatchSnapshot();
  });
});
