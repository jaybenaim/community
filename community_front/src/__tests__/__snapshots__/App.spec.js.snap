// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile Component App has a Map component  1`] = `
<section
  style="height: 100vh; width: 100%;"
/>
`;

exports[`MyProfile Component App has a MyProfile component  1`] = `
<div
  class="cont container"
/>
`;

exports[`MyProfile Component App has a SearchPage component  1`] = `
<div
  class="container container-fluid"
/>
`;

exports[`Nav Component App has a MyProfile component  1`] = `
<nav
  class="nav navbaru"
>
   
</nav>
`;
